### ISSUE TITLE
Should provide better validation output

### ISSUE THREAD

**Issue by DALDEI** [alenkacz/gradle-json-validator#4]
Thanks for this great plugin.
I ran it on a test JSON file and it produces only this output:

```
:validateCatalog FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':validateCatalog'.
> #: 7 schema violations found


```

That is better then nothing, but frustrating saying there are '7 schema violations' but not any indication of what they are, line #, object, property nothing ... 
Even the filename would be nice :)

**Comment by alenkacz** [comment]
@DALDEI that is not that helpful, I agree. I will look into this today in the afternoon and hopefully release new version :-) thanks for the feedback

**Comment by alenkacz** [comment]
@DALDEI improved this and released 1.0.0 - it was actually possible to get better message from the plugin, you just had to run the task with --info or --stacktrace

But that is probably not that user friendly, now it should print out better messages.

Let me know if it still does not work for you
